features/appmgt/org.wso2.carbon.appmgt.store.feature/src/main/resources/store/extensions/assets/mobileapp/pages/assets.jag

```diff
@@ -137,24 +137,26 @@ require('/modules/store.js').exec(function (ctx) {
                 var getNextassetSet = function () {
                     var bamService = Packages.org.wso2.carbon.appmgt.impl.dao.AppMDAO;
                     var bamServiceObj = new bamService();
-                    log.debug("Object initialized successfully : org.wso2.carbon.appmgt.impl.dao.AppMDAO ");
+                    if (log.isDebugEnabled()) {
+                        log.debug("Object initialized successfully : org.wso2.carbon.appmgt.impl.dao.AppMDAO ");
+                    }
 
                     var result = bamServiceObj.getAppsByHitCount(userName, (paging.start + index), 12, config.uiActivityPublishDataSource);
-                    log.debug("query executed successfully to get the assets hit count");
-
+                    if (log.isDebugEnabled()) {
+                        log.debug("query executed successfully to get the assets hit count");
```

Comment:
Always write log message as sentence. Starts with capital and use full stops as necessary.